dpctl/tensor/_manipulation_functions.py

```diff
@@ -102,3 +120,26 @@ def squeeze(X, axes=None):
         return X
     else:
         return dpt.reshape(X, new_shape)
+
+
+def broadcast_to(X, shape):
+    """
+    broadcast_to(X: usm_ndarray, shape: tuple or list) -> usm_ndarray
+
+    Broadcast an array to a new shape; returns the broadcasted
+    array as a view.
+    """
+    if not isinstance(X, dpt.usm_ndarray):
+        raise TypeError(f"Expected usm_ndarray type, got {type(X)}.")
+
+    new_array = np.broadcast_to(
```

Comment:
It may seem that calls to `np.broadcast_to` are not needed. `new_array.ndim` is just `len(shape)` and `new_array.shape` is just normalized `shape`. However, the call also validates input parameter `shape`. Perhaps this warrants a comment.